escheduler-alert/pom.xml

```
@@ -102,6 +102,17 @@
             <artifactId>escheduler-dao</artifactId>
         </dependency>
 
+        <dependency>
```

jar version please use parent pom.xml
```suggestion
           <dependency>
      <groupId>org.apache.httpcomponents</groupId>
      <artifactId>httpcore</artifactId>
    </dependency>
    <dependency>
      <groupId>org.apache.httpcomponents</groupId>
      <artifactId>httpclient</artifactId>
    </dependency>
```